PorticoExportDom.inc.php

```diff
@@ -152,25 +153,26 @@ private function _buildArticle() : DOMElement {
 			$articleNode->appendChild($articleIdListNode);
 
 			$articleIdNode = $articleIdListNode->appendChild($doc->createElement('article-id', $publisherId));
-			$articleIdNode->setAttribute('pub-id-type', 'publisher');
+			$articleIdNode->setAttribute('pub-id-type', 'publisher-id');
 		}
 
-		// galley links
-		$fileService = Services::get('file');
-		$submissionFileDao = DAORegistry::getDAO('SubmissionFileDAO');
-		foreach ($article->getGalleys() as $galley) {
-			if ($url = $galley->getRemoteURL()) {
-				$selfUriNode = $doc->createElement('self-uri', $url);
-				$selfUriNode->setAttribute('xlink:href', $url);
+		$galleys = $article->getGalleys();
```

Comment:
Here I'm considering that only the first galley is a `self-uri` (a link/document that represents the article).
I understand the other galleys could have the same status, but as I didn't see a way to recognize it, I've used this "fallback" rule.

Reply:
Yes, I think that'll work, thanks! Generally only a single file will ever be used.